janus_core/cli/utils.py

```diff
@@ -17,12 +18,14 @@
     from ase import Atoms
     from typer import Context
 
-    from janus_core.cli.types import TyperDict
+    from janus_core.cli.types import CorrelationKwargs, TyperDict
     from janus_core.helpers.janus_types import (
         MaybeSequence,
         PathLike,
     )
 
+from janus_core.processing import observables
```

Comment:
This should perhaps be moved into `parse_correlation_kwargs`, otherwise I think it and other things it depends on e.g. ase may be imported earlier than we need.